### PR TITLE
#36899 Fill in Document metadata on process link

### PR DESCRIPTION
**Pull request by Klaas-Ritense**

### CODE REVIEW

File: projects/valtimo/plugin/src/lib/plugins/documenten-api/documenten-api-plugin.specification.ts [hunk 1]
```diff
@@ -40,6 +40,24 @@ const documentenApiPluginSpecification: PluginSpecification = {
       bronorganisatie: 'Bronorganisatie RSIN',
       bronorganisatieTooltip:
         'Vul hier het RSIN van de organisatie in die verantwoordelijk is voor de documenten',
+      fileName: 'Bestandsnaam',
+      fileNameTooltip:
+        'De naam van het fysieke bestand waarin de inhoud van het document is vastgelegd, inclusief extensie',
+      confidentialityNotice: 'Vertrouwelijkheids aanduiding',
```
Comment (mritense) [inline projects/valtimo/plugin/src/lib/plugins/documenten-api/documenten-api-plugin.specification.ts:46]: Vertrouwelijkheidsaanduiding

File: projects/valtimo/plugin/src/lib/plugins/documenten-api/documenten-api-plugin.specification.ts [hunk 2]
```diff
@@ -73,6 +91,24 @@ const documentenApiPluginSpecification: PluginSpecification = {
       bronorganisatie: 'Organisation RSIN',
       bronorganisatieTooltip:
         'Enter here the RSIN of the organization responsible for the documents. The RSIN is a dutch identification number for legal entities and partnerships ',
+      fileName: 'File name',
+      fileNameTooltip:
+        'The name of the physical file in which the content of the document is captured, including extension',
+      confidentialityNotice: 'Confidentiality Notice',
```
Comment (mritense) [inline projects/valtimo/plugin/src/lib/plugins/documenten-api/documenten-api-plugin.specification.ts:97]: Confidentiality level

File: projects/valtimo/plugin/src/lib/plugins/documenten-api/documenten-api-plugin.specification.ts [hunk 3]
```diff
@@ -73,6 +91,24 @@ const documentenApiPluginSpecification: PluginSpecification = {
...
+      confidentialityNoticeTooltip:
+        'Indication of the extent to which the document is intended for public access',
+      inputTitle: 'Titel',
```
Comment (mritense) [inline projects/valtimo/plugin/src/lib/plugins/documenten-api/documenten-api-plugin.specification.ts:100]: Title

File: projects/valtimo/plugin/src/lib/plugins/documenten-api/documenten-api-plugin.specification.ts [hunk 4]
```diff
@@ -73,6 +91,24 @@ const documentenApiPluginSpecification: PluginSpecification = {
...
+      inputTitleTooltip: 'The name by which the document is formally known',
+      inputDescription: 'Description',
+      inputDescriptionTooltip: 'A generic description of the content of the document',
+      openbaar: 'Public',
+      beperkt_openbaar: 'Restricted public',
+      intern: 'Internal',
+      zaakvertrouwelijk: 'Business confidential',
```
Comment (mritense) [inline projects/valtimo/plugin/src/lib/plugins/documenten-api/documenten-api-plugin.specification.ts:107]: Case confidential

File: projects/valtimo/plugin/src/lib/plugins/documenten-api/documenten-api-plugin.specification.ts (hunk 1 above)
Comment (Klaas-Ritense) [reply projects/valtimo/plugin/src/lib/plugins/documenten-api/documenten-api-plugin.specification.ts]: changed

File: projects/valtimo/plugin/src/lib/plugins/documenten-api/documenten-api-plugin.specification.ts (hunk 3 above)
Comment (Klaas-Ritense) [reply projects/valtimo/plugin/src/lib/plugins/documenten-api/documenten-api-plugin.specification.ts]: changed

File: projects/valtimo/plugin/src/lib/plugins/documenten-api/documenten-api-plugin.specification.ts (hunk 4 above)
Comment (Klaas-Ritense) [reply projects/valtimo/plugin/src/lib/plugins/documenten-api/documenten-api-plugin.specification.ts]: changed

File: projects/valtimo/plugin/src/lib/plugins/documenten-api/documenten-api-plugin.specification.ts (hunk 2 above)
Comment (Klaas-Ritense) [reply projects/valtimo/plugin/src/lib/plugins/documenten-api/documenten-api-plugin.specification.ts]: changed it everywhere

**Pull request merged**